api/illumio/cloud/k8sclustersync/v1/k8s_info.proto

```diff
@@ -322,6 +346,7 @@ message Policy {
 // Flow exported by the CNI plugin in the cluster.
 message SendKubernetesNetworkFlowsRequest {
   oneof request {
+    Keepalive keepalive = 3;
```

Comment:
I don't understand why you'd need to partially pull changes from unmerged PR #155.
Revert this.
```suggestion
```